apps/web/src/components/UserDropdown.tsx

```diff
@@ -0,0 +1,37 @@
+import { useAuthContext } from './AuthContexts';
+import downArrowIcon from '@assets/img/down_arrow.svg';
```

Comment:
we have the fontawesome icon package in the project, any chance of looking in there for icons before adding extra assets? disregard if you already did!